internal/provisioner/plain/controllers/bundledeployment_controller.go

```diff
@@ -46,6 +46,7 @@ import (
 	"sigs.k8s.io/yaml"
 
 	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+	bundleLib "github.com/operator-framework/rukpak/internal/bundle"
```

Comment:
nit: drop the import alias and just use the `bundle` package name in the code.

if i'm missing another import already called `bundle` that requires this alias, i'd suggest `bundlelib` all lowercase. `bundleLib` makes mind mind drift to "that's a variable" rather than "that's a package".

Reply:
actually BundleReploymentReconciler.loadBundle has `bundle` as an arg. IMO that param name makes sense, so I'm keeping `bundleLib`